modules/misc/news.nix

```
@@ -1873,6 +1873,12 @@ in
           'accounts.email.accounts.<name>.msmtp.enable' is true.
         '';
       }
+      {
+        time = "2021-03-01T13:47:00+00:00";
+        message = ''
+          Respect the environment variable NO_COLOR as per https://no-color.org/
```

Perhaps something like
```suggestion
          The home-manager tool now respects the 'NO_COLOR' environment variable
          as per https://no-color.org/.
```
If you want you can also add an entry in the release notes.

---

Done in 3ccf8d8. What release notes do you want me to add it to? 21.05?

---

Added to 21.05 at the bottom of the list in c518dd7